DSP binaries for IQ8275-EVK & IQ9075-EVK platforms

vkraleti opened the pull request:
IQ8275-EVK & IQ9075-EVK  platforms use the same DSP binaries that of existing platforms. To avoid duplication, update `config.txt` to link those to the existing paths as appropriate.

vkraleti committed:
config.txt: link IQ8275-EVK DSP bins to share QCS8300-RIDE paths

Both IQ8275-EVK and QCS8300-RIDE platforms use the same DSP binaries.
To avoid duplication, update `config.txt` to link IQ8275-EVK DSP binary
paths to the existing QCS8300-RIDE binaries.

Signed-off-by: Viswanath Kraleti <viswanath.kraleti@oss.qualcomm.com>

vkraleti committed:
config.txt: link IQ9075-EVK DSP bins to share SA8775P-RIDE paths

Both IQ9075-EVK and SA8775P-RIDE platforms use the same DSP binaries.
To avoid duplication, update `config.txt` to link IQ9075-EVK DSP binary
paths to the existing SA8775P-RIDE binaries.

Signed-off-by: Viswanath Kraleti <viswanath.kraleti@oss.qualcomm.com>

lumag commented on config.txt:
```diff
@@ -46,3 +46,8 @@ Install: qcs8300/Qualcomm/QCS8300-RIDE	gdsp0	gdsp0-DSP.AT.1.0.1-00125.5-LEMANS-1
 Install: qcs615/Qualcomm/QCS615-RIDE	cdsp	cdsp-CDSP.VT.2.2.c4-00019-SM6150-1
 
+# IQ8275 EVK
+Link: qcs8300/Qualcomm/QCS8300-RIDE	qcs8300/Qualcomm/IQ8275-EVK
```
Could you please change this to link individual DSP subdirs? There is no guarantee that these platforms will be airways identical (think about sensors files or ACDB).

vkraleti replied:
Updated please check.

lumag reviewed (changes requested):
Obviously running `make check` isn't required when sending a PR. But please fix it anyway.

vkraleti committed:
scripts/check.py: extend script to support 'Link' entries

Both "Install" & "Link" are valid config actions. Update check.py script
to parse 'Link' entries as well in addition to existing 'Install' entries.

Signed-off-by: Viswanath Kraleti <viswanath.kraleti@oss.qualcomm.com>

vkraleti commented:
> Obviously running `make check` isn't required when sending a PR. But please fix it anyway.

Done.

lumag commented on scripts/check.py:
```diff
@@ -84,22 +84,31 @@ def load_whence():
+        patterns = {
+            "empty": re.compile("^#|^$"),
+            "install": re.compile("Install: ([^ \t]+)[ \t]+([^ \t]+)[ \t]+([^ \t]+)\n"),
+            "link": re.compile("Link: ([^ \t]+)[ \t]+([^ \t]+)\n")
```
Nit: I'd prefer three different variables rather than a dictionary.

lumag commented on scripts/check.py:
```diff
@@ -122,6 +131,30 @@ def check_config(data, dirs):
+
+    if dst_path.endswith("/"):
+        sys.stderr.write("config.txt: %d: trailing '/' in %s\n" % (lineno, dst_path))
+        ret = False
```
Also please check that the source for symlink exists
